app/views/developers/index.html.erb

```diff
@@ -12,14 +12,25 @@
   </p>
 </div>
 
-<div data-controller="pagination" data-pagination-visibility-class="invisible" class="mt-16 mx-auto sm:mt-24">
-  <ul id="developers" data-pagination-target="entries" role="list" class="space-y-8 bg-gray-100">
+<div class="mt-16 mx-auto sm:mt-24">
+  <ul id="developers" role="list" class="space-y-8 bg-gray-100">
     <%= render @developers %>
   </ul>
 
-  <div class="bg-gray-100 pt-8">
+  <div class="mt-8">
     <%= pagy_links do %>
       <%== pagy_nav @pagy %>
     <% end %>
   </div>
+
+  <%=turbo_frame_tag "developers-#{@pagy.page}" do %>
```

Comment:
Please add a space after the `=` (3 more times in this file).

Reply:
Ah darn. I thought I caught on to your desired syntax but this slipped through the cracks :(